pkg/coscheduling/cache/cache.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cache
+
+import (
+	"sync"
+
+	gochache "github.com/patrickmn/go-cache"
+	v1 "k8s.io/api/core/v1"
+
+	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
+)
+
+// Cache provides func Get, Set, Delete to operation cache
+type Cache interface {
+	Get(string) *PodGroupSchedulingStatus
+	Set(string, *PodGroupSchedulingStatus)
+	Delete(string)
+}
+
+var _ Cache = &PGSchedulingCache{}
+
+// PGSchedulingCache provide safe function Get, Set, Delete.
+type PGSchedulingCache struct {
```

Comment:
PGSchedulingCache --> PodGroupCache

Reply:
> PGSchedulingCache --> PodGroupCache

It not only includes podGroup,  pod and others also include.  I would  like keep this name